### ISSUE TITLE
Missing 'delete' parameter?

### ISSUE THREAD

**Issue by xbeta** [Oefenweb/ansible-ufw#19]
the official guide has that option: http://docs.ansible.com/ansible/ufw_module.html

**Comment by tersmitten** [comment]
That's true, but I don't how that would fit into this role. Any suggestions?

**Comment by xbeta** [comment]
@tersmitten for example, get on a VPS box, ssh default is already turn on, but removing the default ssh port 22 to a new port?
